utility/src/main/java/org/oppia/android/util/system/OppiaClock.kt

```diff
@@ -28,4 +28,6 @@ interface OppiaClock {
    * [getCurrentTimeMs].
    */
   fun getCurrentDate(): Date = Date(getCurrentTimeMs())
+
+  fun getElapsedRealTime(): Long
```

Comment:
Do we need a new Regex check to enforce using this instead of System.elapsedRealTime?

I also suggest including the 'Ms' unit in the name to make things clearer. 

I'm also curious about the testing story here. How are you planning to handle the returned value here? Will it have its own time mode like getCurrentTimeMs? Will it share the same mode as getCurrentTimeMs?

Reply:
Removed this, so not needed I guess.